### PR TITLE
Allow Review of & Navigation to Communities from the Family screen

### PR DESCRIPTION
**Pull request by donschaefer**
@LarsKemmann - Not sure if you wanted this functionality bundled into the same PR as the other update outlined in #739 or if you wanted them handled separately, but insofar as...

1. This one _may_ be ready for merging (insofar as it accomplishes the desired outcome relatively quickly given the temporary nature of the change being requested), and
2. I think I'm going to need to pick your brain for some additional context for the remaining work (re: displaying referrals)

... I figured I'd put this up for your review.

### CODE REVIEW

File: src/caretogether-pwa/src/Families/FamilyScreen.tsx [hunk 1]
```diff
@@ -386,6 +402,23 @@ export function FamilyScreen() {
               </Grid>}
           </Grid>
           <Grid container spacing={0}>
+			<Grid item xs={12}>
```
Comment (LarsKemmann) [inline src/caretogether-pwa/src/Families/FamilyScreen.tsx:405]: Put this higher up on the Family screen, above the row of "Incomplete/Completed/Documents" headers in the grid.

File: src/caretogether-pwa/src/Families/FamilyScreen.tsx [hunk 2]
```diff
@@ -386,6 +402,23 @@ export function FamilyScreen() {
...
+              <h3 style={{ marginBottom: 0 }}>Communities</h3>
+				{familyCommunityInfo?.map(communityInfo => {
+					return (
+						<Card variant="outlined" sx={{ paddingBottom: '1rem' }}>
+							<CardHeader 
+								sx={{ paddingBottom: 0 }}
+								title={communityInfo.community?.name}
+								subheader={communityInfo.community?.description}
```
Comment (LarsKemmann) [inline src/caretogether-pwa/src/Families/FamilyScreen.tsx:413]: The description could be very long and currently is primarily used for the church address/phone/etc. Remove it for now.

File: src/caretogether-pwa/src/Families/FamilyScreen.tsx [hunk 3]
```diff
@@ -386,6 +402,23 @@ export function FamilyScreen() {
               </Grid>}
           </Grid>
           <Grid container spacing={0}>
+			<Grid item xs={12}>
+              <h3 style={{ marginBottom: 0 }}>Communities</h3>
+				{familyCommunityInfo?.map(communityInfo => {
+					return (
+						<Card variant="outlined" sx={{ paddingBottom: '1rem' }}>
```
Comment (LarsKemmann) [inline src/caretogether-pwa/src/Families/FamilyScreen.tsx:409]: Can you add `cursor: pointer` to this as an affordance for desktop users that this is clickable?

File: src/caretogether-pwa/src/Families/FamilyScreen.tsx [hunk 4]
```diff
@@ -386,6 +402,23 @@ export function FamilyScreen() {
...
+							<CardHeader
```
Comment (LarsKemmann) [inline src/caretogether-pwa/src/Families/FamilyScreen.tsx:410]: Can you apply the Communities icon (the one that's used in the main nav menu) to the `avatar` prop, to give this card a helpful icon? Similar to [the "R" in the docs](https://mui.com/material-ui/react-card/#complex-interaction), but I think you can pass the icon without an `<Avatar>` component.

**Review by LarsKemmann — CHANGES_REQUESTED** [review]
Functionality is perfect. Display needs a bit of tweaking (see requested changes) to work well with realistic data.

File: src/caretogether-pwa/src/Families/FamilyScreen.tsx (hunk 4 above)
Comment (donschaefer) [reply src/caretogether-pwa/src/Families/FamilyScreen.tsx]: Icon added

File: src/caretogether-pwa/src/Families/FamilyScreen.tsx (hunk 3 above)
Comment (donschaefer) [reply src/caretogether-pwa/src/Families/FamilyScreen.tsx]: Cursor updated

File: src/caretogether-pwa/src/Families/FamilyScreen.tsx (hunk 2 above)
Comment (donschaefer) [reply src/caretogether-pwa/src/Families/FamilyScreen.tsx]: Description removed

File: src/caretogether-pwa/src/Families/FamilyScreen.tsx (hunk 1 above)
Comment (donschaefer) [reply src/caretogether-pwa/src/Families/FamilyScreen.tsx]: Location updated

**Comment by donschaefer** [comment]
@LarsKemmann - apologies that it took me a while to circle back & wrap this up. All updates have now been made (along with a few minor additional improvements). This should be ready for another check when you get a chance.

**Comment by LarsKemmann** [comment]
I'm so sorry for not getting back to this sooner. The edits look perfect -- merged!

**Pull request merged**